README.md

```diff
@@ -184,14 +188,124 @@ Binaries built by Burrito include a built-in set of commands for performing main
+  * `Build` -  This is the final phase in the build flow, it produces the final wrapper binary with a payload embedded inside.
+  
+  You can add your own steps before and after phases execute. Your custom steps will also receive the build context struct, and can return a modified one to customize a build to your liking.
+
+  An example of adding a step before the fetch phase, and after the build phase:
+
+  ```elixir
+  # ... mix.exs file
+  def releases do
+    [
+      my_app: [
+        steps: [:assemble, &Burrito.wrap/1],
+        burrito: [
+          # ... other Burrito configuration
+          extra_steps: [
+            fetch: [pre: [MyCustomStepModule, AnotherCustomStepModule]],
+            build: [post: [CustomStepAgain, YetAnotherCustomStepModule]]
+          ]
+        ]
+      ]
+    ]
+  end
+  ```
+
+#### Build Targets and Qualifiers
+
+A Burrito build target is a keyword list that contains an operating system, a CPU architecture, and extra build options (called Qualifiers).
+
+Here's a definition for a build target configured for Linux x86-64:
+
+```elixir
+targets: [
+  linux: [os: :linux, cpu: :x86_64]
+]
+```
+
+Build targets can be further customized using build qualifiers. For example, a Linux build target can be configured to use `musl` instead of `glibc` using the following definition:
+
+```elixir
+targets: [
+  linux_musl: [
+    os: :linux,
+    cpu: :x86_64,
+    libc: :musl
+  ]
+]
+```
+
+Build qualifiers are a simple way to pass specific flags into the Burrito build pipeline. Currently, only the `libc` and `local_erts` qualifiers have any affect on the standard Burrito build phases and steps.
+
+Tip: You can use these qualifiers as a way to pass per-target information into your custom build steps.
+
+#### Using Custom ERTS Builds
+
+The Burrito project provides precompiled builds of Erlang for the following platforms:
+
+```elixir
+[os: :darwin, cpu: :x86_64],
+[os: :linux, cpu: :x86_64, libc: :glibc], # or just [os: :linux, cpu: :x86_64]
+[os: :linux, cpu: :x86_64, libc: :musl],
+[os: :windows, cpu: :x86_64]
+```
+
+If you require a custom build of ERTS, you're able to override the precompiled binaries on a per target basis by setting local_erts to the path of your ERTS build:
+
+```elixir
+targets: [
+  linux_arm: [
+    os: :linux,
+    cpu: :arm64,
+    local_erts: "/path/to/my_custom_erts.tar.gz"
+  ]
+]
+```
+
+The `local_erts` value should be a path to a local `.tar.gz` of a release from the Erlang source tree. The structure inside the archive should mirror:
+
+```
+. (TAR Root)
+└─ otp-A.B.C-OS-ARCH
+  ├─ erts-X.Y.Z/
+  ├─ releases/
+  ├─ lib/
+  ├─ misc/
+  ├─ usr/
+  └─ Install
+```
+
+You can easily build an archive like this by doing the following commands inside the (official Erlang source code)[https://github.com/erlang/otp]:
```

Comment:
Nitpick: doing -> running

Reply:
Nitpick: inside the official Erlang source code -> from the root directory of the OTP repository